main.go

```diff
@@ -28,57 +28,61 @@ var usageStr = `
 Usage: nats-server [options]
 
 Server Options:
-    -a, --addr, --net <host>         Bind to host address (default: 0.0.0.0)
-    -p, --port <port>                Use port for clients (default: 4222)
-    -n, --name <server_name>         Server name (default: auto)
-    -P, --pid <file>                 File to store PID
-    -m, --http_port <port>           Use port for http monitoring
-    -ms,--https_port <port>          Use port for https monitoring
-    -c, --config <file>              Configuration file
-    -t                               Test configuration and exit
-    -sl,--signal <signal>[=<pid>]    Send signal to nats-server process (stop, quit, reopen, reload)
-                                     <pid> can be either a PID (e.g. 1) or the path to a PID file (e.g. /var/run/nats-server.pid)
-        --client_advertise <string>  Client URL to advertise to other servers
+    -a, --addr, --net <host>                        Bind to host address (default: 0.0.0.0)
+    -p, --port <port>                               Use port for clients (default: 4222)
+    -n, --name, --server_name <server_name>         Server name (default: auto)
```

Comment:
You could have put `--server_name` below maybe so to not have to shift the description of other fields to the right?
Originally we were trying to keep the columns to 80 characters, but to be frank even in main it is already not the case for few options. Maybe we take the opportunity to fix that in that PR?